spec.html

```diff
@@ -845,6 +973,37 @@ <h1>AsyncContext [ @@toStringTag ]</h1>
         <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.</p>
       </emu-clause>
     </emu-clause>
+
+    <emu-clause id="sec-function-properties-of-the-asynccontext-object">
+      <h1>Function Properties of the AsyncContext Object</h1>
+
+      <emu-clause id="sec-asynccontext-wrap">
+        <h1>AsyncContext.wrap ( _target_ )</h1>
+        <p>This function returns a new function which restores the current value of all AsyncContext.Variable values when being invoked.</p>
+
+        <emu-alg>
+          1. If IsCallable(_target_) is false, throw a TypeError exception.
+          1. Let _snapshot_ be AsyncContextSnapshot().
+          1. Let _F_ be ? AsyncContextWrappedFunctionCreate(_target_, _snapshot_).
+          1. Let _L_ be 0.
+          1. Let _targetHasLength_ be ? HasOwnProperty(_target_, *"length"*).
+          1. If _targetHasLength_ is *true*, then
+            1. Let _targetLen_ be ? Get(_target_, *"length"*).
+            1. If _targetLen_ is a Number, then
+              1. If _targetLen_ is *+∞*<sub>𝔽</sub>, set _L_ to +∞.
+              1. Else if _targetLen_ is *-∞*<sub>𝔽</sub>, set _L_ to 0.
+              1. Else,
+                1. Let _targetLenAsInt_ be ! ToIntegerOrInfinity(_targetLen_).
```

Comment:
`targetLenAsInt` probably doesn't need to be a separate variable from `L`.

Reply:
This is a copy paste from https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-function.prototype.bind

Reply:
I know, but in `bind`, `targetLenAsInt` has to be modified to subtract `addCount`, so it makes sense to keep it in a separate variable.

Reply:
Done